src/macros.jl

```diff
@@ -390,6 +390,14 @@ function parse_constraint(_error::Function, args...)
           "       expr1 == expr2\n" * "       lb <= expr <= ub")
 end
 
+# Generic fallback.
+function build_constraint(_error::Function, func,
+        set::Union{MOI.AbstractScalarSet, MOI.AbstractVectorSet})
+    return _error("unable to add the constraint because something is wrong " *
```

Comment:
We can be a bit more specific. The issue is that we don't recognize `func` as a JuMP function.